code/script.js

```diff
@@ -257,47 +258,72 @@ const selectQuestion = () => {
 }
 
 // This function should be invoked when you click on 'Find Out' button.
-const checkQuestion = () => {
-  const { category, value } = currentQuestion
-  
+  const checkQuestion = () => {
+  const {category, value} = currentQuestion
 
   // Compare the currentQuestion details with the secret person details in a different manner based on category (hair/eyes or accessories/others).
   // See if we should keep or remove people based on that
   // Then invoke filterCharacters
-  let keep = false
 
   if (category === 'hair' || category === 'eyes') {
-    keep = value === secret[category];
+    if (value === secret.hair || value === secret.eyes) {
```

Comment:
Because you already defined category === 'hair' || category === 'eyes'
in the line 268. A shorter way to write this is
if (value === secret.category )